src/Core/SettingsChangesHistory.h

```diff
@@ -81,10 +81,12 @@ namespace SettingsChangesHistory
 static std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges> settings_changes_history =
 {
     {"23.1", {{"input_format_json_read_objects_as_strings", 0, 1, "Enable reading nested json objects as strings while object type is experimental"},
+              {"input_format_json_defaults_for_missing_elements_in_named_tuple", false, true, "Allow missing elements in JSON objects while reading named tuples by default"},
               {"input_format_csv_detect_header", false, true, "Detect header in CSV format by default"},
               {"input_format_tsv_detect_header", false, true, "Detect header in TSV format by default"},
               {"input_format_custom_detect_header", false, true, "Detect header in CustomSeparated format by default"},
-              {"input_format_json_defaults_for_missing_elements_in_named_tuple", false, true, "Allow missing elements in JSON objects while reading named tuples by default"}}},
+              {"input_format_json_defaults_for_missing_elements_in_named_tuple", false, true, "Allow missing elements in JSON objects while reading named tuples by default"},
```

Comment:
Seems like a bad merger with master, `input_format_json_defaults_for_missing_elements_in_named_tuple` is duplicated